src/App.tsx

```diff
@@ -1,204 +1,137 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
 import { UserWarning } from './UserWarning';
+import { Todo } from './types/Todo';
+import { Section } from './Components/Todolist';
+import { TodosFooter } from './Components/TodosFooter';
+import { client } from './utils/fetchClient';
 
-const USER_ID = 0;
+const USER_ID = 'https://mate.academy/students-api/todos?userId=11910';
 
 export const App: React.FC = () => {
+  const [inputValue, setInputValue] = useState('');
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterType, setFilterType] = useState('all');
+  const [isLoadingError, setIsLoadingError] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    // Функция для загрузки данных с сервера
+    const fetchTodos = async () => {
+      try {
+        // throw new Error();
+        const response = await client
+          .get<Todo[]>('/todos?userId=11910');
```

Comment:
it's better to specify a function in the separate module of your app and call it this way: `client.getAll()`